buildspec-linux-clang-3.8.yml

```diff
@@ -7,14 +7,15 @@ env:
 
 phases:
   install:
+    runtime-versions:
```

Comment:
Runtime versions only works with the standard images, but none of the standard images works with clang-3.8.

Reply:
OK, removed that change

Reply:
(please re-review)

Reply:
Clang-3.8 is running (again) now, but weirdly it doesn't show up in github.